de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5554220_sys_gh6298_Update_AD_Org_ID_InProcessesToSearch.sql

```diff
@@ -0,0 +1,18 @@
+-- Backup the table AD_Process_Para, Even though only the rows containing 'AD_Org_ID' will be modified
+CREATE TABLE backup.ad_process_para_gh6298_ad_org_id_as_search AS
+    (
+        SELECT *
+        FROM ad_process_para
+    )
+;
+
+
+-- Unconditionally update the `ad_reference_id` to `Search`
+UPDATE ad_process_para
+SET --
+    ad_reference_id = (SELECT ad_reference_id FROM ad_reference WHERE name = 'Search'),
+    UpdatedBy=0,
+    Updated=now()
```

Comment:
```suggestion
    Updated=`2020-03-10 17:01:56.767453+01`
```
use a hardcoded date string that will be the same in the SQL and the DB

Reply:
done